FindmallocMC.cmake

```diff
@@ -96,8 +93,14 @@ if(mallocMC_ROOT_DIR)
     string(REGEX MATCH "([0-9]+)" mallocMC_VERSION_PATCH
                                 ${mallocMC_VERSION_PATCH_HPP})
 
+    # mallocMC variables ########################################################
+    #
+    set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ; mallocMC_INCLUDE_DIRS)
```

Comment:
:+1: I agree. Would be nice if the syntax had been a bit more restrictive from the start ;)